cirq/sim/clifford/clifford_simulator.py

```diff
@@ -366,14 +379,26 @@ def apply_single_qubit_unitary(self, op: 'cirq.Operation'):
         phase_shift = u[max_idx] / applied_unitary[max_idx]
         self.ch_form.omega *= phase_shift
 
-    def _apply_H(self, qubit: ops.Qid):
+    def _apply_H(self, qubit: int):
         self.tableau._H(qubit)
         self.ch_form._H(qubit)
 
-    def _apply_S(self, qubit: ops.Qid):
+    def _apply_S(self, qubit: int):
         self.tableau._S(qubit)
         self.ch_form._S(qubit)
 
+    def _apply_X(self, qubit: int):
+        self.tableau._X(qubit)
+        self.ch_form._X(qubit)
+
+    def _apply_Z(self, qubit: int):
+        self.tableau._Z(qubit)
+        self.ch_form._Z(qubit)
+
+    def _apply_Y(self, qubit: int):
+        self.tableau._Y(qubit)
```

Comment:
I think it would be cleaner if _apply_Gate methods accept the actual Qid and internally resolve that to self.qubit_map[Qid] instead of having the caller do that since the qubit map is an internal implementation detail of the state. (thinking in terms of eventually moving the tableau application to the Gate definitions)